loadtest/control/actions.go

```diff
@@ -152,6 +158,7 @@ func LeaveChannel(u user.User) UserActionResponse {
 	return UserActionResponse{Info: "unable to leave, not member of any channel"}
 }
 
+// JoinTeam adds the given user to a random team.
```

Comment:
Same as what happens with joining a channel. We make sure the user is not a member of a team before trying to join.